src/index.js

```diff
@@ -1,58 +1,22 @@
-var wombleList = require('./templates/womble-list.hbs')
-var detailsTemplate = require('./templates/details.hbs')
+import appTemplate from './templates/app.hbs'
 
 document.addEventListener('DOMContentLoaded', function () {
-  var wombles = [
-    { name: 'Orinocco', email: 'orinocco@wimbledoncommon.net', details: 'Tin cans' },
-    { name: 'Tomsk', email: 'tomsk@wimbledoncommon.net', details: 'Plastic bags' },
-    { name: 'Bungo', email: 'bungo@wimbledoncommon.net', details: 'Discarded wombat poop' }
-  ]
-  refreshContent(wombles)
+  const wombles = getWombles()
+  renderPage(wombles)
 })
 
-function refreshContent (wombles) {
-  var div = document.createElement('div')
-  div.innerHTML = wombleList({ wombles: wombles })
-  document.body.appendChild(div)
-  bindEventListeners(wombles)
-}
-
-function bindEventListeners (wombles) {
-  var lis = document.getElementsByTagName('li')
-  for (var i = 0; i < lis.length; i++) {
-    lis[i].addEventListener('click', function (e) {
-      wombleToggle(e.target, wombles)
-    })
+function renderPage (wombles) {
+  const vm = {
+    wombles: wombles
   }
+  const div = document.getElementById('app')
+  div.innerHTML = appTemplate(vm)
 }
 
```

Comment:
I found all this DOM manipulation and event handling a distraction from client-side rendering.